src/components/List.js

```diff
@@ -4,8 +4,8 @@ import { db } from '../lib/firebase';
 import Error from './Error';
 import calculateEstimate from './../lib/estimates';
 import { differenceInDays, addDays } from 'date-fns';
-import './List.css';
-import Checkbox from '@material-ui/core/Checkbox';
+import '../styles/List.css';
```

Comment:
@veeteeran it looks like you moved the List.css file down into a styles directory.
However, I'm not seeing the file deleted/renamed in this PR. Is the old List.css still hanging around?

Also, I'd be curious to know what your thought process was around moving the file. Co-locating your css files with the component that they style is actually a pretty common practice in modern development, as modularization allowed us to split our CSS files up into component-based chunks, instead of one big global CSS file for the whole app. 
Not saying you shouldn't do this necessarily - I'm just curious what your reasoning was.

Reply:
Hi @chrisdopuch we to move our css files into a styles folder since we thought the organization was better? 😄  Just personal preference on our part. It's good to know that it's common practice to keep the css and components together